docs/api/buckets.rst

```diff
@@ -0,0 +1,199 @@
+Working with buckets
+====================
+
+:ref:`Buckets <buckets>` enable the creation of collections handled by
+a group of users.
+
+As soon as there is a need to let people work on the same records set
+or to work with the same collection, the solution is to create a :ref:`bucket <buckets>`.
+
+:ref:`Buckets <buckets>` gives a public unique name to the collection.
```

Comment:
We're currently changing all the layout of the docs and doing another pass on this proposal. The idea is to have a default bucket for each user.